apps/api/tests/api/application-api.test.ts

```diff
@@ -101,10 +101,10 @@ describe('Application API', () => {
 			assert.strictEqual(state, ApplicationStates.DRAFT);
 
 			const stateUpdate = { state: ApplicationStates.INSTITUTIONAL_REP_REVIEW };
-			const reviewRecord = await testApplicationRepo.findOneAndUpdate({ id, update: stateUpdate });
+			const reviewRecordResult = await testApplicationRepo.findOneAndUpdate({ id, update: stateUpdate });
 
-			assert.ok(Array.isArray(reviewRecord) && reviewRecord[0]);
-			assert.strictEqual(reviewRecord[0].state, ApplicationStates.INSTITUTIONAL_REP_REVIEW);
+			assert.ok(reviewRecordResult.success && reviewRecordResult.data[0]);
+			assert.strictEqual(reviewRecordResult.data[0].state, ApplicationStates.INSTITUTIONAL_REP_REVIEW);
```

Comment:
This change is unrelated to logger, it is included in PRs #59 & #138 as well, issue with this test in `main`